decompiler/config/jak3/jak3_config.jsonc

```diff
@@ -102,14 +102,16 @@
   // LEVEL EXTRACTION
   ////////////////////////////
 
-  // turn this on to extract level background graphics data
-  "levels_extract": false,
-  // turn this on if you want extracted levels to be saved out as .obj files
+  // turn this on to extract level background graphics data as .fr3 files in out/<game>/fr3
+  "levels_extract": true,
```

Comment:
We might still want to keep this off by default in Jak 3 for now until we have everything needed for proper level extraction. Right now, the decompiler is probably going to fail when run with this flag.